verocloud/obsidian-hemingway
# Feat: add retext plugins

### Requests from:

- [ ] Issue: #3 
- [x] Issue: #4 
- [x] Issue: #6 
- [x] Issue: #9 
- [x] Issue: #10 

### What was done?

* Add support for overuse, simplify, assuming and usage plugins.
* Added sentiment level with number and emoji

### How to test it?

* Pick an example sentence from the official docs of each plugins, verify if it shows accordingly to the expected result.
* Verify if user sentiment showing approach is good.
